### PR TITLE
Add unit testing for testcontainers

### PR DESCRIPTION
**Pull request by KyleAure**

### CODE REVIEW

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
Verified unit tests ran: 
Screenshot 2023-05-12 at 9 13 23 AM

**Comment by KyleAure** [comment]
The personal build failed with one existing defect.

**Comment by KyleAure** [comment]
Ready to merge - waiting on review

File: dev/fattest.simplicity/test/componenttest/containers/ArtifactoryImageNameSubstitutorTest.java [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+        //False system property should not append registry
+        setDockerClientStrategy(new UnixSocketClientProviderStrategy());
+        setArtifactoryRegistryAvailable(false);
+        System.setProperty("fat.test.use.artifactory.substitution", "false");
+
+        expected = DockerImageName.parse("openliberty:1.0.0");
+        assertEquals(expected, new ArtifactoryImageNameSubstitutor().apply(expected));
+
+        //True system property should not append registry
```
Comment (gjwatts) [inline dev/fattest.simplicity/test/componenttest/containers/ArtifactoryImageNameSubstitutorTest.java]: Should
`//True system property should not append registry`
be
`//True system property should append registry`
?

**Review by gjwatts — APPROVED** [review]

**Comment by gjwatts** [comment]
#libby

**Pull request merged**